iocore/net/quic/Makefile.am

```diff
@@ -52,6 +52,8 @@ QUICKeyGenerator_impl = QUICKeyGenerator_openssl.cc
 endif
 endif
 
+QLog_impl = qlog/QLogEvent.cc qlog/QlogFrame.cc qlog/QLog.cc
```

Comment:
A filename is slightly wrong and this is why autest fails.

QlogFrame.cc -> QLogFrame.cc (capital L)

Reply:
Fixed